cinema/admin.py

```diff
@@ -0,0 +1,3 @@
+from django.contrib import admin
+
+# Register your models here.
```

Comment:
The `Movie` model is not registered with the Django admin site. Consider adding `admin.site.register(Movie)` to manage the model through the admin interface. Make sure to import the `Movie` model from `cinema.models`.